packages/webpack/src/executors/dev-server/dev-server.impl.ts

```diff
@@ -83,7 +82,6 @@ export async function* devServerExecutor(
       map(({ baseUrl, stats }) => {
         return {
           baseUrl,
-          emittedFiles: getEmittedFiles(stats),
```

Comment:
This is not used at all, and excluding it removes the need for the `getEmittedFiles` function in `lib`.